dashboard/src/components/DetailedView/DetailedView.js

```diff
@@ -87,39 +89,48 @@ const DetailedView = ({ allPollutants = [] }) => {
       <div className="detailed-container">
         <div className="list-and-detail-container">
           <div className="pollutant-select-container">
-            <PollutantList
-              pollutants={pollutants}
-              onPollutantSelect={id => {
-                const currPollutant = pollutants.filter(pollutant => Number(pollutant.id) === Number(id));
-                setSelectedPollutant(currPollutant[0]);
-              }}
-            />
-
-            <div className="pollutant-details-container">
-              <PollutantDetails pollutant={selectedPollutant} />
-            </div>
+            <PollutantList onPollutantSelect={setSelectedPollutant} />
+            <PollutantDetails selectedPollutant={selectedPollutant} />
             <div className="search-box detail-serach-box">
-              <select value={searchType} onChange={handleChange}>
-                {searchOptions.map(option => (
-                  <option key={option.value} value={option.value}>
-                    {option.label}
-                  </option>
-                ))}
-              </select>
-              <input id="cityName" name="cityName" value={searchValue} onChange={e => setSearchValue(e.target.value)} type="text" placeholder="Search..." />
-              <button className="search-btn" onClick={onSearchButtonClick}>Search</button>
+              <div>
+                <input
+                  className="detailed-search-input"
+                  id="cityName"
+                  name="cityName"
+                  value={searchValue}
+                  onChange={(e) => setSearchValue(e.target.value)}
+                  type="text"
+                  placeholder="Search..."
+                />
+                <select
+                  className="detailed-search-select"
+                  value={searchType}
+                  onChange={handleChange}
+                >
+                  {searchOptions.map((option) => (
+                    <option key={option.value} value={option.value}>
+                      {option.label}
+                    </option>
+                  ))}
+                </select>
+              </div>
+
+              <button
+                className="search-btn detail-search-btn"
+                onClick={onSearchButtonClick}
+              >
+                Search
+              </button>
```

Comment:
Are the search input layouts the same between the DetailedView and MainView?

Reply:
Yeah, but we don't need that for the historical view. All we gotta do for the historical view is search for historical data on all 12 pollutants in the list for a specific date range.

Reply:
Right! We have the separate form for the historical view.